### DISCUSSION TITLE
audio guest book on raspberry 3b

### DISCUSSION THREAD

**Discussion by mmichelt** [nickpourazima/rotary-phone-audio-guestbook#68] [Q&A]
I am trying to run the audio guest book on Raspberry 3b.

- Installation of version 1.0.2 went fine.
- I am trying to use the audio jack available on the Raspberry 3b - with a simple headset (old Samsung)
- I was hoping to get it working like that - but I am not able to hear any audio.

**aplay information:**
**_admin@audio-guestbook:~ $ aplay -l_**
**** List of PLAYBACK Hardware Devices ****
card 0: Headphones [bcm2835 Headphones], device 0: bcm2835 Headphones [bcm2835 Headphones]
  Subdevices: 8/8
  Subdevice #0: subdevice #0
  Subdevice #1: subdevice #1
  Subdevice #2: subdevice #2
  Subdevice #3: subdevice #3
  Subdevice #4: subdevice #4
  Subdevice #5: subdevice #5
  Subdevice #6: subdevice #6
  Subdevice #7: subdevice #7
card 1: vc4hdmi [vc4-hdmi], device 0: MAI PCM i2s-hifi-0 [MAI PCM i2s-hifi-0]
  Subdevices: 1/1
  Subdevice #0: subdevice #0

**_admin@audio-guestbook:~ $ aplay -L_**
null
    Discard all samples (playback) or generate zero samples (capture)
default
hw:CARD=Headphones,DEV=0
    bcm2835 Headphones, bcm2835 Headphones
    Direct hardware device without any conversions
plughw:CARD=Headphones,DEV=0
    bcm2835 Headphones, bcm2835 Headphones
    Hardware device with all software conversions
sysdefault:CARD=Headphones
    bcm2835 Headphones, bcm2835 Headphones
    Default Audio Device
dmix:CARD=Headphones,DEV=0
    bcm2835 Headphones, bcm2835 Headphones
    Direct sample mixing device
hw:CARD=vc4hdmi,DEV=0
    vc4-hdmi, MAI PCM i2s-hifi-0
    Direct hardware device without any conversions
plughw:CARD=vc4hdmi,DEV=0
    vc4-hdmi, MAI PCM i2s-hifi-0
    Hardware device with all software conversions
sysdefault:CARD=vc4hdmi
    vc4-hdmi, MAI PCM i2s-hifi-0
    Default Audio Device
hdmi:CARD=vc4hdmi,DEV=0
    vc4-hdmi, MAI PCM i2s-hifi-0
    HDMI Audio Output
dmix:CARD=vc4hdmi,DEV=0
    vc4-hdmi, MAI PCM i2s-hifi-0
    Direct sample mixing device

So I set the Alsa HW Mapping to : plughw:0,0

Any suggestion/help is welcome.

**Comment by nickpourazima** [comment]
Hey @mmichelt , any chance you've tried this already?
```bash
# in a terminal
raspi_config
```
Select 7 Advanced Options, then Audio.
Select "Force 3.5mm ('headphone') jack".
Then "OK".

**Comment by mmichelt** [comment]
Soory for last reply - I was busy with other projects.

`sudo raspi-config`

does not show any option 7

![Clipboard01](https://github.com/user-attachments/assets/8ba6a4f8-f607-4f6a-ab12-babdaf1ed4b7)

under "1 system options" and "S2 Audio" I have the option

![Clipboard02](https://github.com/user-attachments/assets/5bbcc40e-7116-4c5c-87ae-6f1af883502a)

Looks like it does not change anything.

**Reply by mmichelt** [reply]
and option "6- Advanced Options" does not show any audio related items

**Comment by mmichelt** [comment]
Never mind - I used a Pi Zero and it works fine.
